lib/storage/adapters/fs/index.js

```diff
@@ -59,6 +60,13 @@ FSAdapter.SCHEMA_FIELD_DEFAULTS = {
 };
 
 /**
+	Inherit common prototype behaviours for generating and retrying filenames
+	from the keystone-storage-namefunctions package
+*/
+FSAdapter.prototype._getFilename = prototypeMethods.getFilename;
+FSAdapter.prototype._retryFilename = prototypeMethods.retryFilename;
```

Comment:
In the `prototypeMethod.getFilename` function it expects a function `this.retryFilename` not `this._retryFilename`. See: https://github.com/keystonejs/keystone-storage-namefunctions/commit/dc3ef5cc0ea632dac7b2f48a6104c21da8033566